### ISSUE TITLE
short cut fot "the hand", the movement tool

### ISSUE THREAD

**Issue by JUANDI2004** [OpenBoard-org/OpenBoard#1112]
hello! I am a new user on the aplication and I love it! I love how everything could do there with a tablet and a pen. But i had a problem which i consider could help a lot and I DON'T KNOW  if it is in the aplication. I going to describe the problem with the format :D
some thing to have into consideration:
I don't know how to code, so i just could give you some references i used in the past.
I don't know English very well, I apologize some misspell, grammar mistakes, and rude expresions :)

**Is your feature request related to a problem? Please describe.**
No. The sitation i want to add is that the tool of moving along the canvas "the hand" do not have a shortcut. That is hard because if i use a graphic table. usually i use drawing programs and the point is: with the drawing programs you could change and modify the shortcuts. that is super useful because you could do almost everything with the keyboard. In conclusion, as the program does not have the shortcut and you cannot change it, I have to move the arm every time I want to move a bit. 

**Describe the solution you'd like**
I do not know anything about code or something related. the solution, in my consideration is to add a function to change the short cuts and to add short cuts for the most of the functions of the program. I consider that could improve a lot the work flow and the movility through the aplication. I think that putting the opiton of changing all the shortcut of the main tools should improve everything!
I should recoment checking a drawing program to see how it works and if is useful, you should check "krita" which is an open source drawing program. 

**Describe alternatives you've considered**
._.

**Additional context**

Thank you so much! i really hope my ideas help <3
Greetings from colombia <3

**Comment by letsfindaway** [comment]
Starting with OpenBoard 1.7.2 you will have the possibility to configure the shortcuts for actions in the `OpenBoardUser.config` file. This version is likely to be released next month.

You can then add something like this to the configuration file:

```ini
[Shortcut]
actionHand=Ctrl+#, 0, 0
```

The `actionHand` line defines `Ctrl+#` as shortcut. The `, 0 , 0` must be present. It can be used to additionally assign extra mouse keys or buttons on a tablet pen to the action.

But again: all this is only available starting with the next release.

**Comment by ValkCore** [comment]
Is there any way I can use right click instead of a shortcut?

**Comment by letsfindaway** [comment]
> Is there any way I can use right click instead of a shortcut?

No. Mouse buttons can currently not be used to trigger such actions. But to navigate on the page you can use the mouse wheel to scroll up and down and `Alt` + mouse wheel to scroll left and right. `Ctrl` + mouse wheel will zoom.

**Comment by anoudGi** [comment]
I actually cannot find a file titled 

> OpenBoardUser.config

I only have 

> OpenBoard.config

, and adding

```
[Shortcut]
actionHand=Ctrl+#, 0, 0
```

doesn't change anything. I've OpenBoard's latest version (Version 1.7.3). @letsfindaway

**Comment by kaamui** [comment]
Hi @anoudGi,

I'll update the website in the next days. New version of OpenBoard is 1.7.5. You can find it in the downloads page [here](https://github.com/OpenBoard-org/OpenBoard/wiki/Downloads).

For the user config file, you can find it : 

- on Windows : `%localappdata%/OpenBoard`
- on Linux : `~/.local/share/OpenBoard/`
- on macOS . `~/Library/Application Support/OpenBoard/`

**Comment by Hamido97** [comment]
> I actually cannot find a file titled
> 
> > OpenBoardUser.config
> 
> I only have
> 
> > OpenBoard.config
> 
> , and adding
> 
> ```
> [Shortcut]
> actionHand=Ctrl+#, 0, 0
> ```
> 
> doesn't change anything. I've OpenBoard's latest version (Version 1.7.3). [@letsfindaway](https://github.com/letsfindaway)

OpenBoardUser.config can be found in C:\Users\username\AppData\Local\OpenBoard
